src/borg/archiver/_common.py

```diff
@@ -399,6 +399,40 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
             help="consider last N archives after other filters were applied",
         )
 
+    if oldest_newest:
+        group = filters_group.add_mutually_exclusive_group()
+        group.add_argument(
+            "--oldest",
+            metavar="Nd",
+            type=relative_time_marker_validator,
+            dest="oldest",
+            help="consider archives N-days after the oldest archive's timestamp",
+        )
+        group.add_argument(
+            "--newest",
+            metavar="Nd",
+            type=relative_time_marker_validator,
+            dest="newest",
+            help="consider archives N-days before the newest archive's timestamp",
+        )
+
+    if older_newer:
+        group = filters_group.add_mutually_exclusive_group()
+        group.add_argument(
+            "--older",
+            metavar="Nd",
+            type=relative_time_marker_validator,
+            dest="older",
+            help="consider archives older than N-days ago",
+        )
+        group.add_argument(
+            "--newer",
+            metavar="Nd",
+            type=relative_time_marker_validator,
+            dest="newer",
+            help="consider archives between now and N-days ago",
+        )
+
```

Comment:
Is it "N-days" or rather "N days"?

Reply:
Should we also edit it to say "before/after the latest matched archive" to denote that the date filtering happens after match-archives?

Reply:
I wouldn't add too much details here in the options description, but include that in the main docs.